src/ResourceManager/Network/Commands.Network/ApplicationGateway/RedirectConfiguration/RemoveAzureApplicationGatewayRedirectConfigurationCommand.cs

```diff
@@ -0,0 +1,51 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmApplicationGatewayRedirectConfiguration"), OutputType(typeof(PSApplicationGatewayRedirectConfiguration))]
+    public class RemoveAzureApplicationGatewayRedirectConfigurationCommand : NetworkBaseCmdlet
```

Comment:
@akshaysngupta same comment as the Add cmdlet: I don't believe this cmdlet is necessary since it's just manipulating a property that the user has access to in the `PSApplicationGateway` object being passed into this cmdlet